components/ai_chat/resources/page/components/conversation_list/index.tsx

```diff
@@ -14,12 +14,34 @@ import DataContext from '../../state/context'
 import ContextMenuAssistant from '../context_menu_assistant'
 import { getLocale } from '$web-common/locale'
 import SiteTitle from '../site_title'
+import CodeBlock from '../code_block'
 
 const SUGGESTION_STATUS_SHOW_BUTTON: mojom.SuggestionGenerationStatus[] = [
   mojom.SuggestionGenerationStatus.CanGenerate,
   mojom.SuggestionGenerationStatus.IsGenerating
 ]
 
+const buildHTML = (str: string) => {
```

Comment:
This isn't really building "HTML". Perhaps `buildMessage`? Also why not use `function` here?

Reply:
resolved